src/appshell/view/dockwindow/docktoolbar.cpp

```diff
@@ -25,9 +25,13 @@
 
 using namespace mu::dock;
 
-static const QString TOOLBAR_QSS = QString("QToolBar { background: %1; border: 0; padding: 0; }");
+static const qreal TOOLBAR_GRIP_WIDTH(32);
+static const qreal TOOLBAR_GRIP_HEIGHT(36);
 
-static const qreal MOVING_BUTTON_WIDTH(10);
+static const QString TOOLBAR_QSS = QString("QToolBar { background: %4; border: 0; padding: 0; } "
+                                           "QToolBar::handle::horizontal { image: url(\":/view/dockwindow/resources/toolbar_grip_%3_horizontal.svg\"); width: %1px; height: %2px; } "
+                                           "QToolBar::handle::vertical { image: url(\":/view/dockwindow/resources/toolbar_grip_%3_vertical.svg\"); width: %2px; height: %1px; } "
```

Comment:
Can we use ui::IconCode instead of svg?

Reply:
Unfortunately not... 🙁 Qt's options for these handles are very limited. 
We also can't set a different background color when the handle is dragged/pressed.